templates/home/home-fr.html

```diff
@@ -159,21 +141,74 @@ <h2>Votre gouvernement</h2>
 		</div>
 	</div>
 </section>
-<section class="container gc-features">
-	<h2>Initiatives du gouvernement</h2>
-	<div class="row wb-eqht">
-		<div class="col-sm-6">
-			<div class="well well-sm brdr-rds-0 eqht-trgt">
-				<img class="img-responsive full-width" src="../../components/gc-features/img/initiative-520x200.png" alt=""/>
-				<h3 class="h5"><a class="stretched-link" href="#">[Lien à l’élément en vedette]</a></h3>
-				<p>Brève description de l’élément en vedette.</p>
+
+<section class="gc-features">
+	<div class="container">
+		<h2>Initiatives du gouvernement</h2>
+		<!-- START: Add the list of links to the features to display in basic HTML mode -->
+		<ul class="nojs-show">
+			<li><a href="#">[1. Lien à l’élément en vedette]</a></li>
+			<li><a href="#">[2. Lien à l’élément en vedette]</a></li>
+			<li><a href="#">[3. Lien à l’élément en vedette]</a></li>
+			<li><a href="#">[4. Lien à l’élément en vedette]</a></li>
+			<li><a href="#">[5. Lien à l’élément en vedette]</a></li>
+			<li><a href="#">[6. Lien à l’élément en vedette]</a></li>
+			<li><a href="#">[7. Lien à l’élément en vedette]</a></li>
+			<li><a href="#">[8. Lien à l’élément en vedette]</a></li>
+		</ul>
+		<!-- END -->
+
+		<!-- Add the "data-wb-randomize" attribute to the "col-sm-6" element to randomize the features within each theme -->
+		<div class="row wb-eqht-grd">
+			<div class="col-sm-6" data-wb-randomize='{ "toggle": "hide" }'>
+				<div class="well well-sm eqht-trgt hide">
+					<img src="../assets/home-feature-1-520x200.jpg" alt="">
+					<h3><a class="stretched-link" href="#">[1. Lien à l’élément en vedette]</a></h3>
+					<p>1. Brève description de l’élément en vedette.</p>
+				</div>
+				<div class="well well-sm eqht-trgt hide">
+					<img src="../assets/home-feature-2-520x200.jpg" alt="">
+					<h3><a class="stretched-link" href="#">[2. Lien à l’élément en vedette]</a></h3>
+					<p>2. Brève description de l’élément en vedette.</p>
+				</div>
```

Comment:
```suggestion
				<section class="well well-sm eqht-trgt hide">
					<img src="../assets/home-feature-2-520x200.jpg" alt="">
					<h3><a class="stretched-link" href="#">[2. Lien à l’élément en vedette]</a></h3>
					<p>2. Brève description de l’élément en vedette.</p>
				</section>
```

See my rational in the preceding comment